itou/metabase/management/test_populate_metabase_emplois.py

```diff
@@ -327,11 +327,10 @@ def test_check_inconsistencies(capsys):
         management.call_command("populate_metabase_emplois", mode="data_inconsistencies")
 
     stdout, _ = capsys.readouterr()
-    assert stdout.splitlines() == [
-        "Checking data for inconsistencies.",
-        "timeit: method=report_data_inconsistencies completed in seconds=0.00",
-        "timeit: method=handle completed in seconds=0.00",
-    ]
+    out_lines = stdout.splitlines()
+    assert out_lines[0] == "Checking data for inconsistencies."
+    assert out_lines[1][:64] == "timeit: method=report_data_inconsistencies completed in seconds="
+    assert out_lines[2][:43] == "timeit: method=handle completed in seconds="
```

Comment:
@vperron Plus élégant : `assert out_lines[2].startswith("timeit: method=handle completed in seconds=")`. Evite le `:43` cryptique. Mes 2 cts.